hat/assets/js/apps/Iaso/domains/app/translations/en.json

```diff
@@ -675,6 +675,8 @@
     "iaso.orgUnits.bulkChangeCount": "You are about to change {count} Org units",
     "iaso.orgUnits.confirmMultiChange": "Confirm bulk changes ?",
     "iaso.orgUnits.createdBy": "Created By",
+    "iaso.orgUnits.createSubUnitTypes": "Sub org units types to create",
+    "iaso.orgUnits.createSubUnitTypesInfos": "Enable the creation of the above sub org unit types (mobile)",
```

Comment:
I would replace with "Allow the creation of these sub org unit types (mobile)"

The "above sub org unit types"  is not ideal, since the tooltip is actually next to the relevant input